inject-java/src/main/java/io/micronaut/annotation/processing/GenericUtils.java

```diff
@@ -164,7 +164,7 @@ protected TypeMirror interfaceGenericTypeFor(TypeElement element, Class interfac
      * @param interfaceName The interface
      * @return The generic type or null
      */
-    protected TypeMirror interfaceGenericTypeFor(TypeElement element, String interfaceName) {
+    private TypeMirror interfaceGenericTypeFor(TypeElement element, String interfaceName) {
```

Comment:
Why was this class changed?

Reply:
That was an IntelliJ suggestion while I was investigating the checkstyle issue.  You are right, it doesn't need to be changed.  I'll change it back and resubmit.